### PR TITLE
[REVIEW] init changelog 0.3 [skip ci]

### PR DESCRIPTION
**Pull request by pxLi**
Signed-off-by: Peixin Li <pxli@nyu.edu>

init changelog for 0.3 release w/ `GITHUB_TOKEN=<xxx>  scripts/generate-changelog --releases=0.3,0.2,0.1`

To preview the CHANGELOG: https://github.com/pxLi/spark-rapids/blob/changelog-0.3/CHANGELOG.md.

Please help review the issues and PR, correct if any invalid titles, labels or project info. Thanks!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 ## Release 0.3
 
 ### Features
 |||
 |:---|:---|
+|[#444](https://github.com/NVIDIA/spark-rapids/issues/444)|[FEA] Plugable Cache|
+|[#1158](https://github.com/NVIDIA/spark-rapids/issues/1158)|[FEA] Better documentation on type support|
+|[#57](https://github.com/NVIDIA/spark-rapids/issues/57)|[FEA] Support INT96 for parquet reads and writes|
+|[#830](https://github.com/NVIDIA/spark-rapids/issues/830)|[FEA] commonize the version checking in the integration tests|
+|[#817](https://github.com/NVIDIA/spark-rapids/issues/817)|[FEA] Create integration test build against Dataproc|
```
Comment (revans2) [inline CHANGELOG.md:13]: Why are features also tagged with test or build put into this section?  I am fine if that is what the requirements ask for.  It just feels like they are not something that end users would look for in the Features section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +43,8 @@ Generated on 2020-11-30
 ### Performance
 |||
 |:---|:---|
+|[#1048](https://github.com/NVIDIA/spark-rapids/issues/1048)|[FEA] Benchmark runner should upload summary file and Spark event log to s3|
```
Comment (jlowe) [inline CHANGELOG.md:46]: IMO the performance section of the release notes should highlight changes that directly impact the performance of the product.  Most of the things listed here are not that.

File: CHANGELOG.md (hunk 1 above)
Comment (pxLi) [reply CHANGELOG.md:13]: yeah, thats something we may need to re-adjust the label here, the tool was just generating doc by label and project

image

File: CHANGELOG.md (hunk 2 above)
Comment (pxLi) [reply CHANGELOG.md:46]: @sameerz plz help check the labels here, thx!

**Comment by pxLi** [comment]
updated to latest

File: CHANGELOG.md (hunk 1 above)
Comment (sameerz) [reply CHANGELOG.md]: I reviewed the labels on the issues and removed the `feature` label from those issues that were focused on tests.

File: CHANGELOG.md (hunk 2 above)
Comment (sameerz) [reply CHANGELOG.md]: I asked @pxLi to modify the changelog script to not pick up the `benchmark` label for performance features.

**Review by sameerz — APPROVED** [review]

**Comment by pxLi** [comment]
build

**Pull request merged**